### PR TITLE
qadaptive_avgpool2d benchmarking

### PR DESCRIPTION
**Pull request by z-a-f**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29274 qadaptive_avgpool2d benchmarking**
* #29268 qavgpool benchmarking
* #29257 Adding short tests to qmaxpool benchmarking

Differential Revision: [D18343569](https://our.internmc.facebook.com/intern/diff/D18343569)

### CODE REVIEW

File: benchmarks/operator_benchmark/pt/qpool_test.py [hunk 1]
```diff
@@ -89,8 +119,20 @@ def init(self, N, C, H, W, k, s, p, ceil, contig, dtype):
...
+        super(QAdaptiveAvgPool2dBenchmark, self).setup(N, C, *input_size, dtype,
+                                                       contig)
+
+
+op_bench.generate_pt_test(qadaptive_avgpool2d_short_configs,
+                          QAdaptiveAvgPool2dBenchmark)
 op_bench.generate_pt_test(qmaxpool2d_short_configs, QAvgPool2dBenchmark)
 op_bench.generate_pt_test(qmaxpool2d_short_configs, QMaxPool2dBenchmark)
+
+op_bench.generate_pt_test(qadaptive_avgpool2d_long_configs,
```
Comment (mingzhe09088) [inline benchmarks/operator_benchmark/pt/qpool_test.py]: Concatenate configs.

File: benchmarks/operator_benchmark/pt/qpool_test.py [hunk 2]
```diff
@@ -75,20 +97,29 @@ def forward(self):
...
                                           return_indices=False)
         super(QMaxPool2dBenchmark, self).setup(N, C, H, W, dtype, contig)
 
 
 class QAvgPool2dBenchmark(_QPool2dBenchmarkBase):
-    def init(self, N, C, H, W, k, s, p, ceil, contig, dtype):
-        self.pool_op = torch.nn.AvgPool2d(kernel_size=k, stride=s, padding=p,
-                                          ceil_mode=ceil)
+    def init(self, N, C, H, W, k, s, p, contig, dtype):
+        self.pool_op = torch.nn.MaxPool2d(kernel_size=k, stride=s, padding=p,
```
Comment (mingzhe09088) [inline benchmarks/operator_benchmark/pt/qpool_test.py]: Should this be AvgPool2d? Seems to be a mismatch?

Comment (z-a-f) [reply benchmarks/operator_benchmark/pt/qpool_test.py]: You are right -- I noticed it yesterday, forgot to stack tthe changes :)

**Review by mingzhe09088 — APPROVED** [review]
LGTM!

**Pull request closed without merging**